packages/console/src/consts/index.ts

```diff
@@ -10,3 +10,7 @@ export const appearanceModeStorageKey = 'logto:admin_console:appearance_mode';
 export const profileSocialLinkingKeyPrefix = 'logto:admin_console:linking_social_connector';
 export const requestTimeout = 20_000;
 export const defaultPageSize = 20;
+
+export enum queryParameterKeys {
```

Comment:
<img width="334" alt="image" src="https://user-images.githubusercontent.com/14722250/226232072-039ff8da-124f-404e-a8d8-9f17bfdc057c.png">

maybe we can align naming across packages. both are good to me

Reply:
cool updated

Reply:
should update ui as well?

Reply:
align with ui `searchKeys`